docs/src/examples/components/Button/Types/ButtonExample.shorthand.steps.ts

```diff
@@ -0,0 +1 @@
+export { commonThemesSteps as default } from '../../../screenerStepsUtils'
```

Comment:
this is the current approach for testing `ButtonExample.shorthand.tsx` screenshot in all themes, we plan to simplify it, see ISSUES in Description

Reply:
fixed